pkg/controller/installation/core_controller.go

```diff
@@ -742,26 +748,41 @@ func (r *ReconcileInstallation) Reconcile(ctx context.Context, request reconcile
 	terminating := (instance.DeletionTimestamp != nil)
 	preDefaultPatchFrom := client.MergeFrom(instance.DeepCopy())
 
+	// Changes for updating installation status conditions
+	if request.Name == "calico" && request.Namespace == "" {
+		ts := &operator.TigeraStatus{ObjectMeta: metav1.ObjectMeta{Name: "calico"}}
+		err := r.client.Get(context.TODO(), types.NamespacedName{Name: "calico"}, ts)
```

Comment:
You can use `ctx` in stead of `context.TODO()`

Reply:
Done